xml/net_dhcp.xml

```diff
@@ -675,7 +683,7 @@ fixed-address &wsIip;;
   <sect2 xml:id="sec-dhcp-sle">
    <title>The &productname; version</title>
    <para>
-    To improve security, the &productname; version of the ISC's DHCP server
+    To improve security, the &productname; version of the KEA's DHCP server
```

Comment:
I'm not quite sure if at least the mention of the patch is still accurate, because I don't see any chroot patch in the RPM sources, but maybe it was upstreamed, because ISC dhcpd can still be run in a chroot. But anyway for Kea this definitely isn't true, it doesn't run in a chroot, so I would say just leave this section unchanged.

Reply:
As discussed since we still have the ISC server for sp6/sp7 , retaining this section (will be removed for sles 16)